tunneling/python/app.py

```diff
@@ -0,0 +1,35 @@
+import json
+import urllib
+
+import flask
+import requests
+from flask_cors import CORS
+
+app = flask.Flask(__name__)
+CORS(app)
+
+
+@app.route("/bugs", methods=["POST"])
+def bugs():
```

Comment:
Another suggestion is to wrap the request handler in a try...except to avoid returning 500 for several cases like malformed request data (invalid envelope, network error on the `requests.post` call, etc).

For all error cases we can report the error to Sentry 😄